File: duplicate-assert/python-pytest/XpathUtilTest_testGetValue.py
def test_get_value():
    assert getTextAttributeValue(
        createDetailAST(TokenTypes.STRING_LITERAL, "\"HELLO WORLD\"")
    ) == "HELLO WORLD", "Returned value differs from expected"

    assert getTextAttributeValue(
        createDetailAST(TokenTypes.NUM_INT, "123")
    ) == "123", "Returned value differs from expected"

    assert getTextAttributeValue(
        createDetailAST(TokenTypes.IDENT, "HELLO WORLD")
    ) == "HELLO WORLD", "Returned value differs from expected"

    assert getTextAttributeValue(
        createDetailAST(TokenTypes.STRING_LITERAL, "HELLO WORLD")
    ) != "HELLO WORLD", "Returned value differs from expected"

File: duplicate-assert/CSharpXUnit/CSharpXUnit.Tests/Tests/XpathUtilTest_testGetValue.cs
using Xunit;

public class XpathUtilTest
{
    [Fact]
    public void TestGetValue()
    {
        Assert.Equal("HELLO WORLD", 
            GetTextAttributeValue(CreateDetailAST(TokenTypes.StringLiteral, "\"HELLO WORLD\"")), 
            "Returned value differs from expected");
        
        Assert.Equal("123", 
            GetTextAttributeValue(CreateDetailAST(TokenTypes.NumInt, "123")), 
            "Returned value differs from expected");
        
        Assert.Equal("HELLO WORLD", 
            GetTextAttributeValue(CreateDetailAST(TokenTypes.Ident, "HELLO WORLD")), 
            "Returned value differs from expected");
        
        Assert.NotEqual("HELLO WORLD", 
            GetTextAttributeValue(CreateDetailAST(TokenTypes.StringLiteral, "HELLO WORLD")), 
            "Returned value differs from expected");
    }
}
